bigbluebutton-html5/imports/api/deskshare/server/modifiers/modifyDeskshareStatus.js

```diff
@@ -0,0 +1,27 @@
+import Deskshare from '/imports/api/deskshare';
+import Meetings from '/imports/api/meetings';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
+import { Meteor } from 'meteor/meteor';
+import { Match } from 'meteor/check';
+
+export default function modifyDeskshareStatus(meetingId, deskshareInfo) {
```

Comment:
Could you use `check` instead of the `Meteor.test` and store the selector and modifiers inside a variable so its easier to read you are missing the callback on the upsert function.

I think its a good ideia to try having all the server code looking alike.